app/helpers/application_helper.rb

```diff
@@ -106,4 +106,8 @@ def rubygem_search_field(**kwargs)
       **kwargs
     )
   end
+
+  def markdown_to_html(text)
```

Comment:
this needs to come out of the application helper, we don't want to accidentally introduce a way for unescaped user input to be marked `html_safe`